README.md

```diff
@@ -4,33 +4,39 @@
 ensure our commits, pull requests, and more meet a minimum level
 of quality.
 
-## Setup
+[![Gem Version](https://badge.fury.io/rb/moj-danger.svg)](https://badge.fury.io/rb/moj-danger)
+
+## Add Danger to your project
 
 Enable Danger for a project within the [ministryofjustice
 organization](https://github.com/ministryofjustice).
 
-### Configure Danger to use this repo's Dangerfile
-
-Commit a `Dangerfile` containing linking to this repo:
+1. Add the following to your Gemfile;
```

Comment:
This now assumes the user is running Ruby, which isn't going to be true for a decent number of projects. It'd be good do break the instructions up into everyone, Ruby, and not-Ruby, maybe?